files/zh-cn/web/javascript/reference/global_objects/object/freeze/index.md

```diff
@@ -28,7 +28,7 @@ Object.freeze(obj)
 
 ## 描述
 
-冻结一个对象相当于[阻止其扩展](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Object/preventExtensions)然后将所有现有属性的描述符的 `configurable` 特性更改为 `false`——对于数据属性，将同时把 `writable` 特性更改为 `false`。无法向被冻结的对象的属性中添加或删除任何内容。任何这样的尝试都将失败，可能是静默失败，也可能抛出一个 {{jsxref("TypeError")}} 异常（通常情况下，在{{jsxref("Strict_mode", "严格模式", "", 1)}}中抛出）。
+冻结一个对象相当于[阻止其扩展](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Object/preventExtensions)然后将所有现有[属性的描述符](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Object/defineProperty#描述)的 `configurable` 特性更改为 `false`——对于数据属性，将同时把 `writable` 特性更改为 `false`。无法向被冻结的对象的属性中添加或删除任何内容。任何这样的尝试都将失败，可能是静默失败，也可能抛出一个 {{jsxref("TypeError")}} 异常（通常情况下，在{{jsxref("Strict_mode", "严格模式", "", 1)}}中抛出）。
```

Comment:
adding this without modify the upstream?

https://github.com/mdn/content/blob/a0f6bf6f7d148f368f6965255058df1ed1f43839/files/en-us/web/javascript/reference/global_objects/object/freeze/index.md?plain=1#L10

Reply:
This line already has a link to the properties' descriptor in the content repo's main branch, see https://github.com/mdn/content/blob/bcfc519e56326977d6a5d6bae279393441219b1c/files/en-us/web/javascript/reference/global_objects/object/freeze/index.md?plain=1#L33